src/index.ts

```diff
@@ -1,67 +1,13 @@
-import { WebSocketServer } from "ws";
+import { nanoid } from "nanoid";
 
-import { Command, Tetris } from "./core/Tetris";
+import { Tetris } from "./core/Tetris";
+import { TetrisSocket } from "./websocket/Socket";
+import { TetrisSocketServer } from "./websocket/Server";
 
-const SPACE = " ";
-
-const server = new WebSocketServer({ port: 8080 });
+const server = new TetrisSocketServer({ port: 8080 });
```

Comment:
Similar to https://github.com/NunkuGemu/tetribass-client/pull/561#discussion_r907922354. I'm not entirely sure as it's super necessary? Though we might use other ports at some point perhaps?